guides/pages/how-to-use-yjs-subdocuments.mdx

```diff
@@ -0,0 +1,206 @@
+---
+meta:
+  title: "How to use Yjs subdocuments"
+  description: "Learn how to Yjs subdocuments on client and server"
+---
+
+Liveblocks Yjs supports [subdocuments](https://docs.yjs.dev/api/subdocuments),
+which allow you to nest Yjs documents inside each other. This guide takes you
+through how to use them on client and server.
+
+## When to use subdocuments
+
+Subdocuments are helpful when you have multiple _large_ Yjs documents in the
+same room, and you wish to lazy-load them individually. Each subdocument works
+similarly to a normal Yjs document, allowing you to use
+[shared types](https://docs.yjs.dev/getting-started/working-with-shared-types),
+[awareness](https://docs.yjs.dev/getting-started/adding-awareness), and more.
+
+### Not necessary for multiple text editors
+
+Please note that **subdocuments are not necessary for displaying multiple text
+editors** on one page. For this use case, it’s often best to create a
+[`Y.Map`](https://docs.yjs.dev/api/shared-types/y.map) in your Yjs document, and
```

Comment:
> ### Not necessary for multiple text editors
> 
> Please note that **subdocuments are not necessary for displaying multiple text
editors** on one page. For this use case, it’s often best to create a
[`Y.Map`](https://docs.yjs.dev/api/shared-types/y.map) in your Yjs document

Is this a common misconception around subdocuments, or if this is something our customers enquire a lot about subdocuments?

Reply:
I've been asked it a lot of times! Customers have thought subdocs is the only way, but they've only needed two editors at once, and this is easier.